routing/world_graph.hpp

```diff
@@ -19,8 +20,9 @@ class WorldGraph final
   enum class Mode
   {
     SingleMwm,
-    WorldWithLeaps,
-    WorldWithoutLeaps,
+    LeapsOnly,
```

Comment:
```
  enum class Mode
  {
    SingleMwm,  // Mode for building a route within single mwm.
    LeapsOnly,  // Mode for building a cross mwm route contains of only leaps. In case of start and
                // finish they (start and finish) will be connected with all transition segments of
                // their mwm with leap (fake) edges.
    LeapsIfPossible,  // Mode for build cross mwm and single mwm routes. In case of cross mwm route
                      // if they are neighboring mwms the route will be made without leaps.
                      // If not the route is made with leaps for intermediate mwms.
    NoLeaps,  // Mode for building route and getting outgoing/ingoing edges without leaps anyway.
  };
```